ipm_library/test/test_ipm.py

```diff
@@ -279,18 +258,30 @@ def test_map_point_invalid_plane_exception():
     """Check InvalidPlaneException is raised if a plane is invalid, i.e. a=b=c=0."""
     ipm = IPM(tf2.Buffer(), CameraInfo())
     with pytest.raises(InvalidPlaneException):
-        ipm.map_point(Plane(), Point2D(), Time(), '')
+        ipm.map_point(Plane(), Point2D())
 
 
 def test_map_points_invalid_plane_exception():
     """Check InvalidPlaneException is raised if a plane is invalid, i.e. a=b=c=0."""
     ipm = IPM(tf2.Buffer(), CameraInfo())
     with pytest.raises(InvalidPlaneException):
-        ipm.map_points(Plane(), np.array([]), Time(), '')
+        ipm.map_points(Plane(), np.array([]))
 
 
 def test_camera_info_not_set():
     """Check CameraInfoNotSetException is raised if camera info is not set."""
     ipm = IPM(tf2.Buffer())
     with pytest.raises(CameraInfoNotSetException):
-        ipm.map_point(Plane(), Point2D(), Time(), '')
+        ipm.map_point(Plane(), Point2D())
+
+
+def test_map_point_current_time_used_when_time_parameter_is_not_provided():
+    ipm = IPM(tf2.Buffer(), camera_info)
+    point = ipm.map_point(Plane(coef=[0.0, 0.0, 1.0, -1.0]), Point2D())
+    assert point.header.stamp == Time()
```

Comment:
I think `Time()` is the zero time, not the current one. But if you pass the zero time to tf it takes the latest available transform (!= now) for the given chain.

Reply:
I thought zero time meant current time, but you're right, it means the latest available transform.

Reply:
What is with this check? I think it is a bit misleading. It still passes, but afaik only because both are in the same frame, so no transformation happens and the zero time from the default value is not replaced with the stamp of the latest transform.